sale_blanket_order/models/blanket_orders.py

```diff
@@ -483,6 +472,9 @@ def _compute_amount(self):
         default=False,
         help="Technical field for UX purpose.",
     )
+    pricelist_item_id = fields.Many2one(
+        comodel_name="product.pricelist.item", compute="_compute_pricelist_item_id"
+    )
 
     def name_get(self):
```

Comment:
This should be updated as per https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0 

> If you are overriding name_get method, you should now override _compute_display_name instead, adding the possible new fields in the depends of the method. See https://github.com/odoo/odoo/pull/122085 for more details.